### DISCUSSION TITLE
How to add more contexts

### DISCUSSION THREAD

**Discussion by csejourne** [jarun/nnn#1709] [Q&A] [answered]
Hello, I'd like to use nnn with 10 contexts, juste like I use my window manager. I associated in my mind specific number to specific activities, and therefore would like 10 contexts. I didnt see any plugin that does that, is there a simple way I can get more contexts? I check in nnn manual but there is no option in there. Is it possible to simply recompile changing a variable from 4 to 10 ?

Cheers

**Comment by N-R-K** [comment, marked as answer]
You can compile with `make O_CTX8=1` to get 8 context. There's no option for 10.
